build_tools/circle/linting.sh

```diff
@@ -161,3 +161,14 @@ then
     echo $bad_deprecation_property_order
     exit 1
 fi
+
+# Check for default doctest directives ELLIPSIS and NORMALIZE_WHITESPACE
+
+doctest_directive="$(find . -type f \( -name "*.py" -o -name "*.rst" \) -exec grep -nw -E "# doctest: \+(ELLIPSIS|NORMALIZE_WHITESPACE)" {} \;)"
+
+if [ ! -z "$doctest_directive" ]
+then
+    echo "Default doctest directives found:"
```

Comment:
Let's make it clearer what the issue is:
```suggestion
    echo "ELLIPSIS and NORMALIZE_WHITESPACE doctest directives are enabled by default, but were found in:"
```

Reply:
Thanks, I wasn't sure how much information to give.

Reply:
I expect the message will be rarely seen, which is only more reason to provide detail for when it surprises someone!